<launch>
    <arg name="Robot_ip" default="192.168.1.151" />
 
    <node pkg="my_cobot" name="gripper_node" type="gripper_node" output="screen">
        <param name="robot_ip" type="str" value="$(arg Robot_ip)"/>
    </node>
</launch>